spec/rspec/mocks/any_instance_spec.rb

```diff
@@ -123,6 +123,11 @@ def private_method; :private_method_return_value; end
               expect(klass.new.one.plus(1)).to eq(2)
               expect(klass.new.one.plus(2)).to eq(3)
             end
+
+            it 'can use a chain of methods to perform an expectation' do
```

Comment:
We've specced that elsewhere, this is specifically for demonstrating `expect_any_instance_of` works :/